app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharecterView.kt

```diff
@@ -28,26 +38,26 @@ class CharecterView(activity: MainActivity) {
     fun showToastNoItemToShow() {
         val activity = activityRef.get()
         if (activity != null) {
-            val message = activity.baseContext.resources.getString(R.string.message_no_items_to_show)
+            val message = activity.baseContext.resources.getString(com.puzzlebench.clean_marvel_kotlin.R.string.message_no_items_to_show)
             activity.applicationContext.showToast(message)
 
```

Comment:
try to not leave more than one empty line

Reply:
Done! thank´s